### DISCUSSION TITLE
instruction of WSL 1

### DISCUSSION THREAD

**Discussion by azinsharaf** [yuk7/ArchWSL#320] [Q&A] [answered]
I have to use WSL 1 because of using Apple Silicon. I know that WSL 1 support for ArchWSL has been discontinued but there are some workaround provided in the documentation. So my question is when i can't launch the Arch WSL instance (see screenshot below) how i can use pacman and downgrade the packages?!

![image](https://github.com/yuk7/ArchWSL/assets/10101468/2d1e8250-7b1e-4d91-aa38-483451f5a62a)

**Comment by azinsharaf** [comment]
a more descriptive error in power shell:
![image](https://github.com/yuk7/ArchWSL/assets/10101468/31e42c52-4aad-4364-a92c-c563ca69d918)

**Reply by azinsharaf** [reply]
@yuk7 do you have any idea about this error? I did some of the  workarounds i found like setting the arch as default distro in wsl but no luck.

**Reply by yuk7** [reply, marked as answer]
ArchWSL is based on Arch Linux not Arch Linux ARM.
This is not supporting Apple Silicon(ARM64).
This only supports x86_64 architecture. (Intel/AMD/VIA etc.)

**Reply by azinsharaf** [reply]
thanks for answering.
